### PR TITLE
Update glossary.yml

### PR DESCRIPTION
**Pull request by jananiharan**
Added definition for command.

## Author: 

Janani Hariharan

## Language: 

English

## Terms defined:

command

## Editor

Assign the PR based on the language to the following person:

| Language   | Person                       |
|:----------:|:----------------------------:|
| Afrikaans  | @jsteyn, @elletjies          |
| Arabic     | @BatoolMM                    |
| English    | @baileythegreen, @zkamvar    |
| French     | @fmichonneau                 |
| Japanese   | @tomkellygenetics, @masamiy  |
| Portuguese | @beatrizmilz                 |
| Spanish    | @ian-flores                  |

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -1379,6 +1379,11 @@
       ([REPLs](#repl)), including the [Unix shell](#shell), record history and allow
       users to play back recent commands.
 
+- slug: command
+  en:
+    term: "command"
+    def: >
+      An instruction to a computer program to perform a specific task.
```
Comment (baileythegreen) [inline glossary.yml:1386]:
```suggestion
      An instruction telling a computer program to perform a specific task.
```

**Review by baileythegreen — CHANGES_REQUESTED** [review]
Minor change.

File: glossary.yml (hunk 1 above)
Comment (jananiharan) [reply glossary.yml:1386]: Looks good to me!

**Review by baileythegreen — APPROVED** [review]

**Comment by elletjies** [comment]
@all-contributors please add @jananiharan for translation

**Pull request merged**